src/VirtoCommerce.ExperienceApiModule.Core/Schemas/ModuleSettingsType.cs

```diff
@@ -0,0 +1,13 @@
+using GraphQL.Types;
+using VirtoCommerce.ExperienceApiModule.Core.Queries;
+
+namespace VirtoCommerce.ExperienceApiModule.Core.Schemas;
+
+public class ModuleSettingsType : ObjectGraphType<ModuleSettings>
+{
+    public ModuleSettingsType()
+    {
+        Field(x => x.ModuleId, nullable: false);
+        Field<ListGraphType<ModuleSettingType>>("settings");
```

Comment:
change to 
`<NonNullGraphType<ListGraphType<NonNullGraphType<ModuleSettingType>>>>`
"settings" should not be null